### DISCUSSION TITLE
fast forward feature

### DISCUSSION THREAD

**Discussion by byzuzayli** [KevinBytesTheDust/CVAmp#298] [Q&A]
YouTube keeps jumping 10 seconds forward, can't we turn this off? Or can't we add an on/off feature to the user interface?

**Comment by KevinBytesTheDust** [comment]
Are you referring to the github or the [kofi version](https://blueloperlabs.ch/supporter/wf)? Do you notice adverse effects with the jumps?

**Comment by byzuzayli** [comment]
YouTube does not see it as a view because it processes for the same amount of time. The watch time does not count either. Constantly fast forwarding robotically

**Comment by byzuzayli** [comment]
I want to get the Kofi version of the program. I want to support a reliable developer, but very small adjustments or additional features can be made regarding YouTube viewing. @KevinBytesTheDust

**Reply by KevinBytesTheDust** [reply]
There was a reason why I introduced this, but I can't remember. I'll remove it in the next version and we'll see. I've put it on my to-do list. Thank you for speaking up!
